src/modules/comments-counter.js

```diff
@@ -0,0 +1,6 @@
+const countComments = (node) => {
+  const nodes = [...node.children];
+  return nodes.length;
+};
+
+export default countComments;
```

Comment:
- [OPTIONAL] Great Job on the project 👍  . It would be nice to show the number of comments with each title on the homepage to achieve perfection. Cheers 😃

Reply:
Thank you. We will try our best to implement that. :+1: